src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super.setColor(color);

```

Comment:
```suggestion
        super(color);
```

Reply:
Why we can't use setter?

Reply:
you can, but it is a bad practice, you have to initialize an object using a constructor